test/property/local_watcher/index.js

```diff
@@ -71,8 +86,12 @@ async function step (state, op) {
     case 'mv':
       try {
         // XXX fs-extra move can enter in an infinite loop for some stupid moves
```

Comment:
We should remove this comment.

Reply:
I'd prefer to keep it. Someone that see that fs-extra used a lot in this file, and not here, may want to change it, and the mistake is hard to detect.

Reply:
OK. I'm fine with that.